vendor/golang.org/x/crypto/acme/acme.go

```diff
@@ -44,7 +44,7 @@ import (
 
 const (
 	// LetsEncryptURL is the Directory endpoint of Let's Encrypt CA.
-	LetsEncryptURL = "https://acme-v01.api.letsencrypt.org/directory"
+	LetsEncryptURL = "https://acme-v02.api.letsencrypt.org/directory"
```

Comment:
@guillep2k this line is one of the changes that support the v1->v2 upgrade

Reply:
OK, I missed that in the PR description list. 😅